### PR TITLE
Changes needed to use production generated data

### PR DESCRIPTION
**Pull request by kamaldodiya**
Production generated data uses genome_uuid for accessing files and other assets.

The PR is to use production generated data using genome_uuid. 

Review App : http://241-test-gb-data.review.ensembl.org

### CODE REVIEW

File: backend-server/app/model/datalocator.py [hunk 1]
```diff
@@ -24,13 +24,13 @@ class AccessItem(object):
...
-        "jump": "jump/{genome}/jump.ncd",
+        "contigs": "{genome}/contigs.bb",
+        "transcripts": "{genome}/transcripts.bb",
+        "gc": "{genome}/gc.bw",
+        "jump": "{genome}/jump.ncd",
         # "seqs": "seqs/{genome}/{chromosome}",
-        "chrom-hashes": "common_files/{genome}/chrom.hashes.ncd",
-        "chrom-sizes": "common_files/{genome}/chrom.sizes.ncd",
+        "chrom-hashes": "{genome}/chrom.hashes.ncd",
+        "chrom-sizes": "{genome}/chrom.sizes.ncd",
```
Comment (azangru) [inline backend-server/app/model/datalocator.py:33]: Note: we will have to move files on AWS S3 after we merge this PR

File: backend-server/config/sources-s3.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [source.apple]
 driver = "s3"
-url = "http://ensembl-2020-gb-flatfiles.s3.eu-west-2.amazonaws.com/upload-20220829/"
+url = "http://ensembl-2020-gb-flatfiles.s3.eu-west-2.amazonaws.com/upload-2023-10-19/"
```
Comment (azangru) [inline backend-server/config/sources-s3.toml:3]: This is the new directory on AWS that I created for flat file trees. It has only human %gc track there at the moment.

File: backend-server/config/species-list.toml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 [species.grch38]
-path = "homo_sapiens_GCA_000001405.28"
+path = "a7335667-93e7-11ec-a39d-005056b38ce3"
 best_name = "a7335667-93e7-11ec-a39d-005056b38ce3"
 other_names = [
```
Comment (kamaldodiya) [inline backend-server/config/species-list.toml:4]: Is this relevant now ?

Comment (azangru) [reply backend-server/config/species-list.toml:4]: "Other names"? I don't think so. But Andres will remove them when he gets to removing the dependency of species-list.toml

**Review by veidenberg — APPROVED** [review]

**Pull request merged**